src/DIRAC/Core/scripts/dirac_install_db.py

```diff
@@ -12,24 +14,40 @@ def main():
     _, args = Script.parseCommandLine()
 
     # Script imports
-    from DIRAC import gConfig
+    from DIRAC.ConfigurationSystem.Client.Helpers.CSGlobals import useServerCertificate
+    from DIRAC.Core.Security.ProxyInfo import getProxyInfo
     from DIRAC.FrameworkSystem.Client.ComponentInstaller import gComponentInstaller
     from DIRAC.FrameworkSystem.Utilities import MonitoringUtilities
 
+    user = "DIRAC"
+
     gComponentInstaller.exitOnError = True
     gComponentInstaller.getMySQLPasswords()
     for db in args:
         result = gComponentInstaller.installDatabase(db)
         if not result["OK"]:
-            print(f"ERROR: failed to correctly install {db}", result["Message"])
-            continue
+            gLogger.error(f"ERROR: failed to correctly install {db}", result["Message"])
```

Comment:
Very minor suggestion:

```suggestion
            gLogger.error(f"Failed to correctly install {db}", result["Message"])
```